draft-ietf-rats-architecture.md

```diff
@@ -844,40 +835,89 @@ a different format.
 
 # Freshness {#freshness}
 
-It is important to prevent replay attacks where an attacker replays
-old Evidence or an old Attestation Result that is no longer correct.
-To do so, some mechanism of ensuring that the Evidence and Attestation
-Result are fresh, meaning that there is some degree of assurance
-that they still reflect the latest state of the Attester, and that any Attestation
-Result was generated using the latest Appraisal Policy for Evidence.
+It is important to prevent old Evidence or old Attestation Results that are no longer correct to be assessed as valid. Replay attacks are an important reason to do that. More importantly, the validity of Conceptual Messages, such as Evidence or Attestation Results, are the basis for trustworthy remote attestation procedures, in general.
```

Comment:
Please break lines into shorter source lines, like in the old text, to make diffs easier to review.

Reply:
Done in b0488b2 for the whole Freshness section.